tox.ini

```diff
@@ -105,7 +105,7 @@ commands_pre =
   distro: pip install {toxinidir}/opentelemetry-distro {toxinidir}/opentelemetry-distro
   instrumentation: pip install {toxinidir}/opentelemetry-instrumentation
 
-  getting-started: pip install -e {toxinidir}/opentelemetry-instrumentation -e {toxinidir}/opentelemetry-python-contrib/instrumentation/opentelemetry-instrumentation-requests {toxinidir}/opentelemetry-python-contrib/util/opentelemetry-util-http -e {toxinidir}/opentelemetry-python-contrib/instrumentation/opentelemetry-instrumentation-flask
+  getting-started: pip install -e {toxinidir}/opentelemetry-instrumentation -e {toxinidir}/opentelemetry-python-contrib/instrumentation/opentelemetry-instrumentation-requests {toxinidir}/opentelemetry-python-contrib/util/opentelemetry-util-http -e {toxinidir}/opentelemetry-python-contrib/instrumentation/opentelemetry-instrumentation-wsgi -e {toxinidir}/opentelemetry-python-contrib/instrumentation/opentelemetry-instrumentation-flask
```

Comment:
I am not sure why it isn't failing but tracecontext test also needs wsgi as dependency?

Reply:
That's a good point... i wonder if it's using the tox cache which would have the dependency in it already, maybe this would explain why this other tox target didn't fail before either

Reply:
Actually looking at the tracecontext job, i don't see it requiring the flask instrumentation, am i missing something?

Reply:
It doesn't need flask instrumentation. It requires [`OpenTelemetryMiddleware`](https://github.com/open-telemetry/opentelemetry-python/blob/main/tests/w3c_tracecontext_validation_server.py#L49) from `opentelemetry-instrumentation-wsgi`.

Reply:
To give more context `OpenTelemetryMiddleware` doesn't exist in `util-http` and ideally this import should fail;
https://github.com/open-telemetry/opentelemetry-python/blob/731f32c4ace615d743b424860741334d76369780/tests/w3c_tracecontext_validation_server.py#L33-L49

 no idea how it is working, as mentioned earlier probably some cache but needs some investigation.

Reply:
> To give more context `OpenTelemetryMiddleware` doesn't exist in `util-http` and ideally this import should fail;
> https://github.com/open-telemetry/opentelemetry-python/blob/731f32c4ace615d743b424860741334d76369780/tests/w3c_tracecontext_validation_server.py#L33-L49
> 
> no idea how it is working, as mentioned earlier probably some cache but needs some investigation.

I guess this file is not being imported at all when running `tox`, checking that...

Reply:
Thanks for catching this @lonewolf3739, fixed and found one more missed reference in the docs

Reply:
> > To give more context `OpenTelemetryMiddleware` doesn't exist in `util-http` and ideally this import should fail;
> > https://github.com/open-telemetry/opentelemetry-python/blob/731f32c4ace615d743b424860741334d76369780/tests/w3c_tracecontext_validation_server.py#L33-L49
> > 
> > no idea how it is working, as mentioned earlier probably some cache but needs some investigation.
> 
> I guess this file is not being imported at all when running `tox`, checking that...

Yes, that file is not part of any `tox`-related test.

Reply:
that file gets run as part of the tracecontext job